### PR TITLE
Add default properties for each background

### PR DESCRIPTION
**Pull request by jaydhulia**
Add default camera properties, position, rotation, scale and hint for each of our background assets

### CODE REVIEW

File: configs/dev-aws-config.yaml [hunk 1]
```diff
@@ -131,3 +131,152 @@ scene_data:
...
+      - 0.425
+      - 0.425
+      - 0.425
+    43:
+      - 0.425
+      - 0.425
+      - 0.425
+  hints:
+    - "You can use your mouse to pan around, and use the arrow keys / WASD to move around"
+    - "If you are seeing this page and aat a ren't the admin, please contact Calgary ConnecTeen to let them know that a solution is missing :)"
```
Comment (ahmedhamodi) [inline configs/dev-aws-config.yaml:282]: Typo :)

File: configs/dev-config.yaml [hunk 2]
```diff
@@ -130,3 +130,152 @@ scene_data:
...
+      - 0.425
+      - 0.425
+      - 0.425
+    43:
+      - 0.425
+      - 0.425
+      - 0.425
+  hints:
+    - "You can use your mouse to pan around, and use the arrow keys / WASD to move around"
+    - "If you are seeing this page and aat a ren't the admin, please contact Calgary ConnecTeen to let them know that a solution is missing :)"
```
Comment (ahmedhamodi) [inline configs/dev-config.yaml:281]: Same here

**Review by ahmedhamodi — COMMENTED** [review]
Small typo. Otherwise code looks good to me. I'll test it later today if Dhruvin doesn't beat me to it.

File: configs/dev-aws-config.yaml (hunk 1 above)
Comment (jaydhulia) [reply configs/dev-aws-config.yaml]: Fixed!

File: configs/dev-config.yaml (hunk 2 above)
Comment (jaydhulia) [reply configs/dev-config.yaml]: Fixed!

Comment (ahmedhamodi) [reply configs/dev-config.yaml]: I think there is still a typo. `...page and aat aren't the...`. The "aat" shouldn't be there.

**Review by ahmedhamodi — DISMISSED** [review]
One you remove the `aat` text from both the configs, this looks good to me.

File: configs/dev-config.yaml (hunk 2 above)
Comment (jaydhulia) [reply configs/dev-config.yaml]: Updated

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**